version.tf

```
@@ -1,7 +1,13 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = "~> 0.13.0"
```

Guess that in this case the module is also compatible with Terraform v0.12.x. So this should then be:
```suggestion
  required_version = ">= 0.12.0"
```